Resources/Prototypes/DeltaV/Entities/Objects/Specific/Justice/gavel.yml

```diff
@@ -0,0 +1,18 @@
+- type: entity
+  parent: BaseItem
+  id: Gavel
+  name: gavel
+  description: A hardwood mallet made to keep order in the court.
+  components:
+    - type: Sprite
+      sprite: DeltaV/Objects/Specific/Justice/gavel.rsi
+      layers:
+        - state: icon
+    - type: MeleeWeapon
+      wideAnimationRotation: -90
+      damage:
+        types:
+          Blunt: 5
```

Comment:
Not sure if intentional but this does similar damage to a screwdriver / crowbar. Although i don't know how viable an IRL Gavel is in close-quarters.